docs/dev/table/sql/queries.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Queries"
+title: "SELECT statements"
```

Comment:
Remove `Operations` section as we moved them into separate pages.

Reply:
I will revert `SELECT statements` to `Queries` because Queries contain not only SELECT statement but also VALUES statement